.github/workflows/deploy.yml

```
@@ -14,21 +14,30 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DATABASE_URL: ${{ secrets.DATABASE_URL }}
+      ECR_REPOSITORY_URLS: ${{ secrets.ECR_REPOSITORY_URLS }}
     strategy:
       matrix:
         node-version: [22]
     steps:
     - name: Checkout
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)